ocaml/ocaml5.nix

```diff
@@ -36,6 +36,93 @@ with oself;
 
   httpaf-eio = callPackage ./httpaf/eio.nix { };
 
+
+
+  irmin-eio = buildDunePackage {
+    pname = "irmin";
+
+    version = "2023-08-25+eio";
+
+    src = fetchFromGitHub {
+      owner = "mirage";
+      repo = "irmin";
+      rev = "5eb94d774e505ee9f870bbfdbc799b07439a489d";
```

Comment:
shouldn't this be overriding the entire irmin package? and use `inherit (irmin) src;` here?

Reply:
I thought `irmin-eio` was another package. I get it now.

Can you add a comment saying which branch this commit comes from?